doc/en/logging.rst

```diff
@@ -198,6 +198,16 @@ option names are:
 * ``log_file_format``
 * ``log_file_date_format``
 
+You can call ``set_log_path()`` to customize the log_file path dynamically::
```

Comment:
I think it is enough here to mention that this feature is experimental.

Reply:
Thank you, fixed

Reply:
I think we should add the example hook (that was part of  https://github.com/pytest-dev/pytest/commit/77b0dd3f95693be67528f7afb449eca9a79d026a once this feature is no longer experimental.

Reply:
Thanks @thisch. I am agreed with you. I will take a note for myself about this. Is there a best practice to not forgot this kind of future changes?

Reply:
I would simply create a ticket for this.